tekton/resources/nightly-tests/serviceaccount.yaml

```diff
@@ -0,0 +1,72 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: tekton-test-nightly
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: tekton-test-nightly-trigger-default
+  namespace: default
+subjects:
+- kind: ServiceAccount
+  name: tekton-test-nightly
+  namespace: default
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: tekton-triggers-nightly
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: tekton-test-nightly-trigger-bastion-z
+  namespace: bastion-z
+subjects:
+- kind: ServiceAccount
+  name: tekton-test-nightly
+  namespace: default
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: tekton-triggers-nightly
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: tekton-triggers-nightly-clusterrole
```

Comment:
This name already exists - since it's a cluster role it will fail as duplicate

Reply:
updated to `tekton-test-triggers-nightly-clusterrole`